packages/view/backend/sirius-components-view-builder/src/main/java/org/eclipse/sirius/components/view/builder/generated/BarChartDescriptionBuilder.java

```diff
@@ -0,0 +1,118 @@
+package org.eclipse.sirius.components.view.builder.generated;
+
+/**
+* Builder for BarChartDescriptionBuilder.
+* @generated
+*/
+@SuppressWarnings("checkstyle:JavadocType")
```

Comment:
Fix the generator and the generated code instead

Reply:
IMO, unless we do not want checkstyle running on _*.generated_ package, keeping the _@SuppressWarnings_ is the best solution.